### ISSUE TITLE
Brave Linux doesn't properly respect its darkmode setting or the OS's

### ISSUE THREAD

**Issue by Lid3616** [brave/brave-browser#31129]
## Description 
Issue is with dark mode in normal settings menu. If I open Brave and it's set to dark mode, the window (tabs, url bar) will still be light theme and all websites loaded will be light theme. If I change it to light and then dark and relaunch the browser, it ignores the setting and always launches in light mode.

## Steps to Reproduce

   1. Open Brave
   2. Set to Dark Mode
   3. Relaunch
   4. Check window bars, website with darkmode

## Actual result:

Windows and borders will be light theme
Website will be very bright (light theme)

## Expected result:

My eyes not to hurt
Dark window borders and dark themes on websites

## Reproduces how often: 
Hasn't worked properly at all for me

## Brave version (brave://version info)
Version 1.52.126 Chromium: 114.0.5735.133 (Official Build) (64-bit)](https://brave.com/latest/)

## Version/Channel Information:
Unable to check other channels

## Other Additional Information:

-Linux mint fully upgraded

**Comment by Lid3616** [comment]
UPDATE:

Also tried brave beta and it had the same issues, really love Brave browser and this makes it very hard to use. Currently had to maually install a dark theme and toggle dark mode for websites gloabally each day in order to work around this issue.

**Comment by uint69-t** [comment]
Same for me, can confirm that persists in the last version:
`
1.52.126 Chromium: 114.0.5735.133 (Official Build) (64-bit)
Revision fbfa2ce68d01b2201d8c667c2e73f648a61c4f4a-refs/branch-heads/5735@{#1270}
Linux (Mint) 5.15.0-73-generic #80-Ubuntu SMP Mon May 15 15:18:26 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
`

When i open the setting, it says Dark, if i switch to Light and then back to Dark, it works again, but as soon as i close/open the web browser its back. (Also, i dont have any cookie-setting, i saw a lot of threads about clearing cookies at exit may cause this, but im not clearing cookies)

It was working good around a month ago.

**Comment by uint69-t** [comment]
UPDATE:
I tried running brave with the console to check for logs, i've found that when i click the settings tab:
`[222794:222794:0625/032513.229900:ERROR:CONSOLE(2115)] "[Brave Settings Overrides] Appearance Page cannot find use default theme button template", source: chrome://settings/settings.js (2115)`

**Comment by Tony-Klink** [comment]
Same issue here. I'm on NixOS + home-manager and impermanence module.
My config:
```
{ pkgs, lib, ... }: {
  home.packages = with pkgs; [ brave ];
  
  home = {
  sessionVariables.BROWSER = "brave";
    persistence = {
      "/persist/home/user" = {
        allowOther = true;
        directories = [
          ".config/BraveSoftware"
        ];
      };
    };
  };
  
  xdg.mimeApps.defaultApplications = {
    "text/html" = [ "brave.desktop" ];
    "text/xml" = [ "brave.desktop" ];
    "x-scheme-handler/http" = [ "brave.desktop" ];
    "x-scheme-handler/https" = [ "brave.desktop" ];
  };
}
```

**Comment by stephenjudge** [comment]
Confirmed as the issue is described above occurring on Brave Version 1.52.129 Chromium: 114.0.5735.198 (Official Build) (64-bit) on Xubuntu 22.04.2 LTS

**Comment by Lid3616** [comment]
I should note that for seperate reasons I had to do a full OS reinstall, and I installed the newest Brave Beta and this issue seems to be fixed for me, so it's very possible someone is working on/has made a fix and it's in the pipeline. If they have and it's not blind luck I'd like to leave my thanks here.

**Comment by uint69-t** [comment]
> I should note that for seperate reasons I had to do a full OS reinstall, and I installed the newest Brave Beta and this issue seems to be fixed for me, so it's very possible someone is working on/has made a fix and it's in the pipeline. If they have and it's not blind luck I'd like to leave my thanks here.

full OS reinstall? Im running the last Brave version, maybe a Brave reinstall cleaning some profile files solves the issue?

**Comment by Lid3616** [comment]
> > I should note that for seperate reasons I had to do a full OS reinstall, and I installed the newest Brave Beta and this issue seems to be fixed for me, so it's very possible someone is working on/has made a fix and it's in the pipeline. If they have and it's not blind luck I'd like to leave my thanks here.
> 
> full OS reinstall? Im running the last Brave version, maybe a Brave reinstall cleaning some profile files solves the issue?

Yeah I got some new HD's and a hardware upgrade and wanted everything to be clean, I'm not recommending anyone else reinstall their whole system though lol

https://brave.com/download-beta/

The beta installer is available here, not sure if you're running latest beta or stable, but you can install them side by side

For reference on linux your profile is in "~/.config/BraveSoftware/Brave-Browser/Default" deleting the contents of this folder will delete everything saved in your browser though. If you have the beta installed you should have "~/.config/BraveSoftware/Brave-Browser-Beta/Default" for your profile.

**Comment by emhl** [comment]
brave-browser 1.52.129-1 still has this issue for me. on brave-browser-beta 1.54.84-1 it doesn't happen
seems like it got fixed at some point

**Comment by uint69-t** [comment]
I can confirm that using the latest beta (1.54.84 Chromium: 115.0.5790.40 (Official Build) beta (64-bit)) solves the issue!

**Comment by gknapp** [comment]
Just confirming I'm seeing this issue on Linux Mint 21.1. Toggling in settings manually to restore dark mode is a pain.

Brave: [Version 1.52.129 Chromium: 114.0.5735.198 (Official Build) (64-bit)]

**Comment by edeutscher** [comment]
+1 I have been dealing with this issue for weeks now.  every time i launch it it is in light mode but in the settings it is "dark" and i switch to light then back to dark and it is correct.  this just started around 4 weeks ago so it was a bug introduced recently.  would love it if this got fixed

**Comment by newedgex** [comment]
I'm using brave-bin 1:1.52.130-1 on Arch Linux.
I have tried also rm -rf everything and start from scratch, this sucks pretty badly. I use multiple profiles and having to switch each time to dark is making me want to remove completely brave and use firefox.
Please , have some compassion for vampires! :'D

**Comment by posita** [comment]
It sounds like this issue is a duplicate of both #14685 and #30766?

**Comment by GabrielTMartinez** [comment]
ok, I know this issue is a little old, but if you have set Block Fingerprinting to Strict, there's this flag brave://flags/#brave-dark-mode-block that will always inform light mode to websites you visit. If you turn it to Disabled and relaunch it, it solves the issue, at least for websites. Not sure about the windows and borders part, didn't happen for me
